app/client/jsx/EmailForm.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { HttpApi } from './WebAPI.jsx'
+import Config from './Config.jsx'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+    faCircleNotch,
+    faEnvelope,
+    faExclamationCircle
+} from '@fortawesome/free-solid-svg-icons'
+
+
+export default ({ user }) => {
+    const MIN_WORDS = 10
+    const MAX_WORDS = 500
+
+    const STATUS = {
+        sending: {
+            label: 'Your message is being sent...',
+            icon: faCircleNotch
+        },
+        succeeded: {
+            label: 'Message sent!',
+            icon: faEnvelope
+        },
+        failed: {
+            label: `Something went wrong sending your message. Please close this box and try again, or email the moderator directly at ${Config.moderation.moderatorEmails[0]}.`,
+            icon: faExclamationCircle
+        }
+    }
+
+    const httpApi = new HttpApi()
+
+    const [email, setEmail] = useState('')
+    const [message, setMessage] = useState('')
+    const [sent, setSent] = useState(false)
+    const [status, setStatus] = useState('sending')
+
+    const numWords = message.split(/\s+/).length - 1
+    const wordCountClass = numWords < MIN_WORDS || numWords === MAX_WORDS ? 'bad-word-count' : ''
+
+    const emailRegex = new RegExp(/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/)
```

Comment:
how can we as professionals not use the [true email regex](http://www.ex-parrot.com/~pdw/Mail-RFC822-Address.html)